### PR TITLE
feat(2624): add design and modify flow in MASE

### PR DESCRIPTION
**Pull request by lengyel-arpad85**
## Changes proposed in this pull request

solves most of #2624 

- removed / renamed `shoe-shop` page/url to `mock-idp/consent` for it to make more sense 
- added functionality to create a new account
- added edit account functionality & a way to add funds to an account
- added consistent design to pages (see screenshots)
- made modifications to the consent pages, in structure and design

| Screenshots (v2) |   |   |
|--------|--------|--------|
|  |  |  |

_____________

| Screenshots (old) |   |   |
|--------|--------|--------|
|  |  |  |
|  |  |  | 

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [x] Make sure that all checks pass
- [ ] Bruno collection updated

### CODE REVIEW

File: localenv/mock-account-servicing-entity/app/components/Button.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  {
+    variants: {
+      intent: {
+        default:
+          'bg-[#F37F64] hover:bg-[#DA725A] disabled:bg-mercury disabled:text-gray-500 shadow-md text-white',
+        danger:
+          'disabled:bg-red-200 bg-red-500 hover:bg-red-600 shadow-md text-white'
+      },
+      size: {
+        sm: 'px-2 py-1 rounded-md font-medium',
```
Comment (Tymmmy) [inline localenv/mock-account-servicing-entity/app/components/Button.tsx:16]: you could add rounded and font size to general styles, as they are the same for both sizes

**Review by Tymmmy — DISMISSED** [review]

**Comment by mkurapov** [comment]
A few of us took a look at the screenshots, and had some comments:

In general, looks much better, thank you! 
The main thing we wanted to do was to distinguish it a bit more from the Admin UI, just to avoid any confusion for people working through the local playground, between what would be part of Rafiki, or something external.

Some thoughts we had:
- Adding the name of the instance (eg "Happy Life Bank" or "Cloud Nine Wallet") instead of "Mock ASE" , and adding some kind of icon to go along with it, either the wallet icon (like the one in the new test wallet/rafiki.money redesign) or the bank icon
- Adding something like global disclaimer to the footer of the page, explaining that this is just a "Mock ASE" and not for use as a production interface
- Mainly to differentiate from the Rafiki UI, there was an idea was to move the two tabs to the top of the page (at least somewhere other than the left side bar)
- And in general, some minor changes in the color, to contribute to the separation between the mock ASE and Admin UI

I'll start a thread in slack in case someone else had additional thoughts.

**Comment by JoblersTune** [comment]
How about this for a different look:
image
We could have a different background for the wallet ASE using it's wallet svg instead of the banks.

**Comment by JoblersTune** [comment]
Attached are the assets I created:
file:///home/sarah/Downloads/wallet-white.svg
file:///home/sarah/Downloads/wallet-background.png
file:///home/sarah/Downloads/background.png
file:///home/sarah/Downloads/Bank%20Icon%20(1).svg



Colors:

- EBF8FF
- 3533A0
- 242374

**Comment by lengyel-arpad85** [comment]
> A few of us took a look at the screenshots, and had some comments:
> 
> In general, looks much better, thank you! The main thing we wanted to do was to distinguish it a bit more from the Admin UI, just to avoid any confusion for people working through the local playground, between what would be part of Rafiki, or something external.
> 
> Some thoughts we had:
> 
> * Adding the name of the instance (eg "Happy Life Bank" or "Cloud Nine Wallet") instead of "Mock ASE" , and adding some kind of icon to go along with it, either the wallet icon (like the one in the new test wallet/rafiki.money redesign) or the bank icon
> * Adding something like global disclaimer to the footer of the page, explaining that this is just a "Mock ASE" and not for use as a production interface
> * Mainly to differentiate from the Rafiki UI, there was an idea was to move the two tabs to the top of the page (at least somewhere other than the left side bar)
> * And in general, some minor changes in the color, to contribute to the separation between the mock ASE and Admin UI
> 
> I'll start a thread in slack in case someone else had additional thoughts.

The update:
 
- changed left menubar to top menu
- added the separate logos and background image for each instance
- added instance name to meta title and topmenu
- added infobar to bottom of pages with the disclaimer
- changed colors used (see new screenshots)

**Review by Tymmmy — APPROVED** [review]

**Review by Tymmmy — APPROVED** [review]

File: localenv/mock-account-servicing-entity/app/routes/rates._index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+export const loader = async () => {
+  const rates = config.seed.rates
+  const currencies = Object.keys(rates)
+
+  return json({ currencies, rates })
+}
+
+export default function Rates() {
```
[the remark this answers is not in the document]
Comment (lengyel-arpad85) [reply localenv/mock-account-servicing-entity/app/routes/rates._index.tsx]: I've changed the route for the page to rates/view, so the original route is now accessible by the backend services again

File: localenv/mock-account-servicing-entity/app/routes/accounts.$accountId.tsx [hunk 3]
```diff
@@ -1,53 +1,329 @@
...
+                <div className='w-full p-4 space-y-3'>
+                  <Input
+                    type='number'
+                    name='balance'
+                    label='Amount to add'
+                    placeholder='0'
+                    min={1}
+                    error={response?.errors.addLiquidity.fieldErrors.amount}
+                    onChange={onAmountChange}
+                  />
```
[the remark this answers is not in the document]
Comment (lengyel-arpad85) [reply localenv/mock-account-servicing-entity/app/routes/accounts.$accountId.tsx:175]: zod coerce to bigint does not work for decimal numbers so I've changed that, it now accepts values containing decimals

**Review by Tymmmy — APPROVED** [review]

**Review by mkurapov — APPROVED** [review]

**Pull request merged**